jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy

```diff
@@ -212,10 +316,9 @@ def run_single_it_test(String case_name) {
                                     envVar(key: 'MYSQL_ROOT_PASSWORD', value: "${MYSQL_PSWD}"),
                             ],
                             args: "${MYSQL_ARGS}"),
-                    // mysql 8.0
```

Comment:
did you consult someone before you make this change?

Reply:
this pr is WIP, and I was going to ask someone who know this place during the working day